sdk/tables/azure-data-tables/src/tables_clients.cpp

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#include <azure/data/tables/internal/policies/service_version_policy.hpp>
-#include <azure/data/tables/internal/policies/shared_key_lite_policy.hpp>
-#include <azure/data/tables/internal/policies/shared_key_policy.hpp>
-#include <azure/data/tables/internal/policies/switch_to_secondary_policy.hpp>
-#include <azure/data/tables/internal/policies/tenant_bearer_token_policy.hpp>
-#include <azure/data/tables/internal/policies/timeout_policy.hpp>
-#include <azure/data/tables/internal/serializers.hpp>
-#include <azure/data/tables/tables_clients.hpp>
+#include "azure/data/tables/tables_clients.hpp"
+
```

Comment:
Same question/nit here.

Reply:
new line added by clang formatting not me